setup.py

```diff
@@ -50,7 +50,13 @@
     extras_require=dict(
         tsne=["fitsne"],
         leiden=["leidenalg"],
-        mkl=["mkl"]
+        louvain=["louvain"],
+        scanorama=["scanorama"],
+        gprofiler=["gprofiler-official"],
```

Comment:
I don't think `gprofiler` is in your task card. And we have already removed it from Pegasus.